src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -181,21 +253,15 @@
  * Example of allowed public immutable fields:
  * </p>
  * <pre>
- * public class ImmutableClass
+ * public final class MyClass
  * {
- *   public final ImmutableSet&lt;String&gt; includes; // No warning
- *   public final ImmutableSet&lt;String&gt; excludes; // No warning
- *   public final java.lang.String notes; // No warning
- *   public final BigDecimal value; // No warning
- *
- *   public ImmutableClass(Collection&lt;String&gt; includes, Collection&lt;String&gt; excludes,
- *                BigDecimal value, String notes)
- *   {
- *     this.includes = ImmutableSet.copyOf(includes);
- *     this.excludes = ImmutableSet.copyOf(excludes);
- *     this.value = value;
- *     this.notes = notes;
- *   }
+ * public final java.lang.String notes; // Ok, from immutable class canonical names
```

Comment:
Let's use same java code in all examples. Yes, it might be slightly big example block but user will see a difference.

Reply:
you mean making first properties, immutable, and annotations classes in one single class in all examples?
I thought I'd do this before but I thought it would get too big as an example to explain, so if this what you mean i will change them now.

Reply:
Yes,  we need to combine them.

It works better for users.

But if t see how we can minimize example - do it

Reply:
Can you please look at this error? 
> checkstyle.checkstyle (Job spelling) 
2 hours to solve it, even i deleted all the comments and it still fails.

Reply:
Just apply patch as terminal command
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=13705&view=logs&j=6a833393-c551-5000-0916-5000fd3b86e7&t=bdbbd129-77f0-5d73-ce7d-db58d34c8358&l=38

It is copy-paste, and keep in commit.